Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/MainController.java

```diff
@@ -287,7 +299,7 @@ public void onEvent(Event event) throws Exception {
             UserSessionManager.setMainController(this);
             pagingandbuttons.setVisible(true);
 
-        } catch (Exception e) {
+        } catch (final Exception e) {
             LOGGER.error("\n\n##### Repository NOT available: {}", e.getMessage());
 
             e.printStackTrace();
```

Comment:
Do `LOGGER.error("Repository not available", e);` rather than sending the stack trace to stdout.

Reply:
Done.